### ISSUE TITLE
Create test folder with a demo html page + webpack config

### ISSUE THREAD

**Issue by malinushj** [Va1/browser-sync-webpack-plugin#64]
This could be useful for contributors to test any changes (manually for now) inside the repo, not having to use an external environment.
This folder should be in `.npmignore.`

Automated testing I think is out of the scope of this project since it's more of a wrapper plugin for Browsersync.
Perhaps some unit tests for the css emissions check?

**Comment by Va1** [comment]
test/demo folder is nice idea! i might work on this in near future.

i am not sure of how can this be unit-tested though

**Comment by malinushj** [comment]
@Va1 It's fine, I think I'll do it, just wrote it down as an issue so we won't forget 😁 
About unit testing, I've seen some other plugins do tests, even intergration tests directly running webpack so it might be possible, but I have to do some deeper checks.

**Comment by Va1** [comment]
@malinushj yeah, i did tests for webpack stuff as part of [other package](https://github.com/Va1/string-replace-loader) developed by me. but i am not sure i understand what we can cover in this package
